add clarifying language about using items instead of item (#172)
radiantearth/stac-api-spec: ogcapi-features/README.md

is outside the scope of STAC API, as the [STAC Item](../stac-spec/item-spec/item-spec.md) is
specified in GeoJSON.

A typical OAFeat will have multiple collections. Simple search for items within a collection can be done
with the resource endpoint `GET /collections/{collectionId}/items`. This endpoint should be exposed via a 
link in the individual collection's endpoint with `rel=items`, as shown in the 
[Example Landing Page diagram](../overview.md#example-landing-page). Note that this relation is `items`, which is

> distinct from the `item` relation defined in STAC for linking to a single Item. Most APIs will not use 

Reviewer: ```suggestion
distinct from the `item` relation defined in STAC for linking to a single Item. The part of the API implementing OAFeat will usually not use 
```

`item` relations directly, but instead rely on 
the collection resource linking to a paginated endpoint returning items through a link relation 
`items`, e.g., `/collections/{collectionId}` has a link with relation `items` linking 

> to `/collections/{collectionId}/items`. However, static catalogs and APIs can be exposed via the same 

Reviewer: ```suggestion
to `/collections/{collectionId}/items`. However, static catalogs and other parts of the API can be combined 
```

> to `/collections/{collectionId}/items`. However, static catalogs and other parts of the API can be combined 

Reviewer: ```suggestion
to `/collections/{collectionId}/items`. However, static catalogs and other parts of the API may contain `item` relations.
```

> endpoints, so APIs may contain `item` relations.

Reviewer: so that APIs that do not implement OAFeat may contain `item` relations.

Reviewer: Seems I made a mistake here. Fixed it, looks good now.

> so that APIs that do not implement OAFeat may contain `item` relations.

Reviewer: ```suggestion
```
